.github/workflows/ci.yml

```diff
@@ -55,6 +55,12 @@ jobs:
           # the backend that uses poll, and is not a public API.
           RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg polling_test_poll_backend
         if: startsWith(matrix.os, 'ubuntu')
+      - run: cargo test
+        env:
+          # Note: This cfg is intended to make it easy for polling developers to test
+          # the backend that uses pipes, and is not a public API.
+          RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg polling_test_epoll_pipe
```

Comment:
Is this really a cfg mainly intended for use in testing?
If not, it may be preferable to remove "\_test\_" and simply state that it is an unstable API.

Reply:
Yes, it should only be used in testing. The pipe fallback is mostly for containers and should happen dynamically.